pyproject.toml

```diff
@@ -41,7 +41,8 @@ dependencies = [
     "nbclassic",
     # pyvo pin can be removed once astroquery's pyvo dep pulls in 1.5.3
     "pyvo>=1.5.3",
-    "s3fs>=2024.10.0"
+    "s3fs>=2024.10.0",
+    "spherical-geometry"
```

Comment:
we're trying to be very intentional about adding additional dependencies, although this one seems to be maintained internally and pretty lightweight.  Do we need any specific version?

Reply:
We don't need a specific version, at the moment. We're also cautious about adding deps, but implementing our own `spherical-geometry` in jdaviz would be strange.